packages/tools/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+- We added compatibility with node 15+ and npm 7+.
+- We added resolutions for `react`, `react-dom` and `react-native` in order to be compatible with node 15+ and npm 7+.
+- We added typing generation support for association widget properties.
+
+### Changed
+- We updated version of `@wdio` and `react-native` libraries.
+
+### Removed
+- We removed `@wdio/sync` from our dependency. 
+
+## BREAKING CHANGES
```

Comment:
Is this the recommend way of indicating breaking changes? 😄

Reply:
This is how we usually do. But we need to update docs as well